Sources/Swift/Integrations/SessionReplay/SentrySessionReplay.swift

```diff
@@ -258,7 +258,11 @@ class SentrySessionReplay: NSObject {
             events.append(contentsOf: touchTracker.replayEvents(from: videoSegmentStart ?? video.start, until: video.end))
             touchTracker.flushFinishedEvents()
         }
-
+        
+        if segment == 0 {
+            events.append(SentryRRWebOptionsEvent(timestamp: video.start, options: self.replayOptions))
```

Comment:
`events` is an array of breadcrumbs AFAICT, can we rename that as well as the types like `SentryRRWebOptionsEvent` -> `SentryRRWebOptionsBreadcrumb` and `SentryRRWebCustomEvent` -> `SentryRRWebCustomEventBreadcrumb` (or s/breadcrumb/info, etc)

Reply:
Events is not an array o breadcrumbs. `Events` is and array or RRWebEvents, among them we have breadcrumbs, video info, and custom events.

Reply:
OK, `events` is really just an array of SentrySerializable-conforming objects and is a mixture of both breadcrumbs and RRWebEvents, it seems. I only looked at the return type for `convertBreadcrumbs` and didn't check out `replayEvents(from:until:)`. Polymorphism between ObjC and Swift is fun :D I still think

Reply:
Not that it really matters for this PR, but here's how I would've named that function so that it's clearer at the callsite what's happening: 
<img width="1699" alt="image" src="https://github.com/user-attachments/assets/7f667283-805e-497e-93d4-5c952e14a3af" />